VMs/metrics/ubuntu16.04/user-data

```diff
@@ -42,6 +43,7 @@ runcmd:
  - sudo add-apt-repository "deb [arch=ARCH] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
  - apt-get update
  - apt-get install -y docker-ce
+ - apt-get install -y linux-tools-`uname -r`
```

Comment:
You could just make this `linux-tools-generic` to have apt dtrt.

Reply:
I saw that in the docs, and tried it inside the VM, and it didn't seem to get me everything I needed - perf complained it was not for the correct kernel version :-( So I ended up with the `uname -r`, which then forced me to put the package as a hand apt call rather than in the package list further up.